### PR TITLE
feat(basemaps): Upgrade vector elt include title in the stac. BM-1017

### PR DESCRIPTION
**Pull request by Wentao-Kuang**
#### Motivation

The new version of vector etl container to which include the fix of road addressing big tiles, and include the title into stac file.

#### Modification

- include title in the etl cmd

#### Checklist

- [ ] Tests updated
- [ ] Docs updated
- [x] Issue linked in Title

### CODE REVIEW

**Review by l0b0 — DISMISSED** [review]

File: workflows/basemaps/vector-etl.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       
       - name: version_basemaps_etl
         description: Version of the Basemaps ETL eks container to use
-        value: 'v1'
```
Comment (blacha) [inline workflows/basemaps/vector-etl.yaml:20]: we shouldnt need to update this? `v1` should match `v1.x.x` ?

Comment (Wentao-Kuang) [reply workflows/basemaps/vector-etl.yaml:20]: Yeah, I just prefer to fix to 1.2 first in case any future changes broken the v1 version.

Comment (blacha) [reply workflows/basemaps/vector-etl.yaml:20]: if there are breaking changes it shouldn't be `v1` it should be made a `v2` ?

File: workflows/basemaps/vector-etl.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ spec:
...
+        value: 'v1.2'
```
Comment (Wentao-Kuang) [inline workflows/basemaps/vector-etl.yaml:20]:
```suggestion
        value: 'v1'
```

**Review by l0b0 — APPROVED** [review]

**Pull request merged**